l10n_it_ddt/data/mail_template_data.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <!--Email template -->
+        <record id="email_template_edi_ddt" model="mail.template">
+            <field name="name">DDT- Send by Email</field>
+            <field name="email_from">
+                ${(object.company_id.email and '%s &lt;%s&gt;' % (object.company_id.name, object.company_id.email) or '')|safe}
+            </field>
+            <field name="subject">${object.company_id.name} (${object.ddt_number})
+            </field>
+            <field name="partner_to">${object.partner_id.id}</field>
+            <field name="email_to">
+                ${(object.partner_id.email and '%s &lt;%s&gt;' % (object.partner_id.name, object.partner_id.email) or '')|safe}
+            </field>
+            <field name="report_template" ref="action_report_ddt"/>
+            <field name="report_name">${(object.ddt_number)}</field>
+            <field name="auto_delete" eval="True"/>
+            <field name="model_id" ref="model_stock_picking_package_preparation"/>
+            <field name="lang">${object.partner_id.lang}</field>
+            <field name="body_html"><![CDATA[
+<p>Dear ${object.partner_id.name},</p>
+<p>Here is your document of
+%if object.date:
+<strong>${object.date},</strong>
+% endif
+with shipping date
+%if object.date_done:
+<strong>${object.date_done}</strong>
+%endif
+from ${object.company_id.name}.
+</p>
+<p>You can reply to this email if you have any questions.</p>
+<p>Thank you.</p>
+]]></field>
+        </record>
+
+        <record id="mail_template_data_notification_email_ddt" model="mail.template">
```

Comment:
Why 2 email templates?

Reply:
Has been split for convenience, the second template contains header and footer and the first the body